### PR TITLE
Step up and added signup

### PR DESCRIPTION
**Pull request by rishibhargava**

### CODE REVIEW

**Comment by rishibhargava** [comment]
@gshriki Please review this request and approve. It is for step-up. We still need to fix the flow but I am not able to load the console app due to slow internet here.

**Comment by rishibhargava** [comment]
I was able to test the step up with only oauth so this is all good. Waiting for review from @gshriki before merging.

**Review by gshriki — APPROVED** [review]

**Pull request merged**
